File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/MedicinalProductManufactured.py
from typing import Optional, List
from base import CodeableConcept
from base import ProdCharacteristic
from base import Quantity
from base import DomainResource
from base import Reference

class MedicinalProductManufactured(DomainResource):
    quantity: Quantity
    ingredient: Optional[List[Reference]] = None
    manufacturer: Optional[List[Reference]] = None
    unit_of_presentation: Optional[CodeableConcept] = None
    manufactured_dose_form: CodeableConcept
    other_characteristics: Optional[List[CodeableConcept]] = None
    physical_characteristics: Optional[ProdCharacteristic] = None

File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/MedicinalProductManufactured.ts
import { CodeableConcept } from "./CodeableConcept";
import { ProdCharacteristic } from "./ProdCharacteristic";
import { Quantity } from "./Quantity";
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";

export type MedicinalProductManufactured = DomainResource & {
    quantity: Quantity;
    ingredient?: Reference[];
    manufacturer?: Reference[];
    unitOfPresentation?: CodeableConcept;
    manufacturedDoseForm: CodeableConcept;
    otherCharacteristics?: CodeableConcept[];
    physicalCharacteristics?: ProdCharacteristic;
};